flags/application.json

```diff
@@ -40,11 +40,10 @@
         "shift": 8,
         "undocumented": true
     },
-    "ALLOW_ACTIVITY_ACTION_SPECTATE": {
-        "description": "Allows the application to enable activity spectating.",
+    "CONTEXTLESS_ACTIVITY": {
+        "description": "Currently unknown.",
```

Comment:
```suggestion
        "description": "This embedded application's activity can be launched without a context.",
```

Reply:
TMK "context" refers to the source of the application (voice chat, message, etc). Haven't looked into the specifics though.